weblate/accounts/forms.py

```diff
@@ -975,7 +979,7 @@ class TOTPDeviceForm(forms.Form):
     )
 
     error_messages = {
-        "invalid_token": gettext_lazy("Entered token is not valid."),
+        "invalid_token": gettext_lazy("The entered token is wrong."),
```

Comment:
```suggestion
        "invalid_token": gettext_lazy("The entered token is has expired."),
```
On the assumption it is know that this token was at one time valid.

Reply:
No, we know it is not valid, nothing else.